app/models/manageiq/providers/autosde/storage_manager/physical_storage.rb

```diff
@@ -13,6 +15,19 @@ def raw_delete_physical_storage
     ext_management_system.class::AutosdeClient.enqueue_refresh(self.class.name, id, ext_management_system.id, task_id)
   end
 
+  def event_where_clause(assoc = :ems_events, storage_systems = nil)
+    case assoc.to_sym
+    when :ems_events, :event_streams
+      if storage_systems && storage_systems != ['']
+        return manipulate_storage_systems(assoc, storage_systems)
+      end
+
+      ["#{events_table_name(assoc)}.physical_storage_id = ?", id]
+    when :policy_events
+      ["target_id = ? and target_class = ? ", id, self.class.base_class.name]
```

Comment:
Super-minor, but capitalize and SQL keywords for consistency.

```suggestion
      ["target_id = ? AND target_class = ? ", id, self.class.base_class.name]
```